src/main/java/frc/robot/commands/elevator/ResetEncoderAtBottom.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.commands.elevator;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.constants.ElevatorConstants;
+import frc.robot.subsystems.Elevator;
+
+public class ResetEncoderAtBottom extends CommandBase {
+  Elevator m_elevator; 
+  /**
+   * Calibrate the elevator by resetting the encoder after it hits the bottom
```

Comment:
You might want to make it clear zeroing the encoder is only part of the calibration prosses